containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -476,7 +476,7 @@ TEST(TEST_CATEGORY, create_mirror_no_init_dynrankview_viewctor) {
         ASSERT_EQ(device_view.size(), mirror_device.size());
         auto mirror_host = Kokkos::create_mirror(
             Kokkos::view_alloc(Kokkos::WithoutInitializing,
-                               Kokkos::DefaultExecutionSpace{}),
+                               Kokkos::DefaultHostExecutionSpace{}),
```

Comment:
Is it advantageous to specify `DefaultHostExecutionSpace{}` vs. the previous version, `DefaultExecutionSpace{}`?  In other words, isn't the `ExecutionSpace` something that should remain generic, vs. a sort of predetermining  `ExecutionSpace` ?  The same question for lines 550 - 552, 558, 586, 594, etc.

Reply:
We create a mirror in a host-accessible space here. Hence, the provided execution space must also be a host execution space.